proto/warehouse/warehouse.proto

```diff
@@ -11,6 +11,8 @@ service Warehouse {
   rpc GetWHUploads (WHUploadsRequest) returns (WHUploadsResponse);
   rpc GetWHUpload (WHUploadRequest) returns (WHUploadResponse);
   rpc TriggerWHUpload (WHUploadRequest) returns (google.protobuf.Empty);
+  rpc TriggerWHUploads (WHUploadsRequest) returns (google.protobuf.Empty);
```

Comment:
so, we would not respond with something like there are no pending files that need to be uploaded? can we use the getpendingstagingfiles func to quickly check that before triggering or something